docs/css/docs.css

```diff
@@ -120,6 +120,33 @@
   margin-bottom: 0 !important;
```

Comment:
You should not use important ideally.

Reply:
It was necessary to use the `!important` here to overwrite  CSS injected by JavaScript in Github Gist HTML

Reply:
cool.